src/styles/card.scss

```diff
@@ -0,0 +1,140 @@
+@use './variables' as *;
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+  margin: 0 auto;
```

Comment:
Consider using the variables for the grid properties like the gap, padding, and margin to maintain consistency and ease of maintenance.